src/java.base/share/native/libjli/java.c

```diff
@@ -525,6 +529,17 @@ JavaMain(void* _args)
     appClass = GetApplicationClass(env);
     NULL_CHECK_RETURN_VALUE(appClass, -1);
 
+    /*  Get launcher helper class. */
+    helperClass = GetLauncherHelperClass(env);
+
+    /*  Validate and fetch main method from application class. */
+    validateMainMethod = (*env)->GetStaticMethodID(env, helperClass,
+                                         "validateMainMethod",
+                                         "(Ljava/lang/Class;)Ljava/lang/reflect/Method;");
+    CHECK_EXCEPTION_NULL_LEAVE(validateMainMethod);
```

Comment:
The comment in line 498-499 needs update as `LoadMainClass` no longer validates the main method.
```
   * Get the application's main class. It also checks if the main
   * method exists.
```

Reply:
Changing